core/globals.h

```diff
@@ -109,6 +109,32 @@
 #    define DYNAMORIO_EXPORT
 #endif
 
+#ifdef WINDOWS
```

Comment:
This seems out of place splitting up the import/export defines here (some above and some below): how about moving it at least below DR_UNS_API

Reply:
Done.